### PR TITLE
AEROGEAR-1630 : Cleaned up the cordova section

### PR DESCRIPTION
**Pull request by sebastienblanc**
@edewit @lfryc Could you review ?

### CODE REVIEW

**Comment by edewit** [comment]
:eyes:

File: docs/guides/aerogear-cordova/CordovaSetup.asciidoc [hunk 1]
```diff
@@ -8,35 +8,11 @@ breadcrumbs-url: /getstarted/guides/
 sub-section-title: Cordova
 section-class: guides
 ---
+== Apache Cordova CLI 
 
-== Setting up your development enivronment to use Apache Cordova
+Cordova comes with a CLI (Command Line Interface) which help starting with link:http://cordova.apache.org/docs/en/4.0.0/guide_cli_index.md.html[Cordova].
```
Comment (edewit) [inline docs/guides/aerogear-cordova/CordovaSetup.asciidoc]: which help starting with / to help you get started with

**Comment by edewit** [comment]
:+1:

**Comment by danielpassos** [comment]
:+1:

**Comment by lfryc** [comment]
:+1: merging

**Comment by lfryc** [comment]
Merged: https://github.com/aerogear/aerogear.org/commit/5f6a3c7c60c14b2e3e83d1a5d8f7f7a20643e49e

**Pull request closed without merging**